src/pre_defined_classes/simple_programs_taking_numeric_user_input/medium/q6/MainTest.java

```diff
@@ -0,0 +1,55 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.medium.q6;
+
+import global.BaseRandomTest;
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class MainTest extends BaseRandomTest {
+    // Parsons
+    public Clause[] testSentence() {
+        TestOption.isInputTest = true;
+        TestOption.defaultInput = "0";
+        return new Clause[]{
+                new StringLiteral("Are computers smarter than humans\\? \\(0 for no, 1 for yes\\)\\: "),
+                new NewLine(),
+                new PlaceHolder(),
+                new StringLiteral("You are "),
+                new PlaceHolder(),
+                new StringLiteral(" percent correct")
+        };
+    }
+
+    public void runMain() {
+        Intelligence.main(new String[0]);
+    }
+
+    @Test
+    public void disbelieveComputer() throws InvalidClauseException {
```

Comment:
The name of this method eventually gets printed out to the user on the website, so it should be something that makes logical/grammatical sense and gives them an idea of what we are testing for.